### PR TITLE
added tauthor's email to msg to paper authors

### PR DESCRIPTION
**Pull request by mohituniyal**

### CODE REVIEW

**Comment by mohituniyal** [comment]
@melisabok I have added the paper number. Also, now the process fn adjusts if there is no abstract added by the author. I found that if there is no abstract, we were sending "undefined" in our emails to authors.

File: openreview/conference/templates/submissionProcess.js [hunk 1]
```diff
@@ -3,10 +3,16 @@ function() {
 
   var SHORT_PHRASE = '';
 
+  var authorMessage = 'Your submission to ' + SHORT_PHRASE + ' has been posted.\n\nSubmission Number: ' + note.number + '\n\nTitle: ' + note.content.title;
+  if (note.content.abstract) {
+    authorMessage += '\n\nAbstract: ' + note.content.abstract;
+  }
+  authorMessage += '\n\nTo view your submission, click here: ' + baseUrl + '/forum?id=' + note.forum + '\n\nIf you are not an author for this submission and want yourself removed, you can do so by reaching out to the person who added you at ' + note.tauthor;
```
Comment (melisabok) [inline openreview/conference/templates/submissionProcess.js]: @zbialecki could you review the wording here please?

Comment (zbialecki) [inline openreview/conference/templates/submissionProcess.js]: I would suggest that we change the wording to "If you are not an author of this submission and would like  to be removed, please contact the author who added you at..."

Comment (mohituniyal) [reply openreview/conference/templates/submissionProcess.js]: thanks for the suggestion. Change has been made.

**Pull request merged**